Navigation rework

sickelap opened the pull request.

sickelap committed:
initial navigation rework (wip)

sickelap committed:
refactor explorer

derneuere committed:
Reset selection on change of select mode

derneuere committed:
Improve the selection color of faces

derneuere committed:
Add new local for confidence

derneuere committed:
Add new local for successful upload

derneuere committed:
Add delete conformation for faces

derneuere commented:
I know it's a draft, but here are a couple of thoughts I had, while looking at it:

I like the new list component on the explorer page. It also makes sense that data visualization is on that page. 

As a user, I would probably expect public photos on the sharing page. 

Face recognition could be on the person page. But maybe it needs a new tab on the side menu, because right now it's pretty much needed in order to label persons and therefore needs to be discoverable. Once we have an automated clustering system for faces in place, we can then move it. 

Also, the auto generated albums are missing :sweat_smile:  We should maybe also rename that to "Events" or "Moments" or something like this.

Vistaus committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (412 of 412 strings)

Translation: Librephotos/LibrePhotos Frontend
Translate-URL: https://hosted.weblate.org/projects/librephotos/librephotos-frontend/nl/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (412 of 412 strings)

Translation: Librephotos/LibrePhotos Frontend
Translate-URL: https://hosted.weblate.org/projects/librephotos/librephotos-frontend/fr/

Commit:
Translated using Weblate (German)

Currently translated at 99.7% (411 of 412 strings)

Translation: Librephotos/LibrePhotos Frontend
Translate-URL: https://hosted.weblate.org/projects/librephotos/librephotos-frontend/de/

sickelap commented:
There are a few things that need to be looked at:
- Mobile menu still hasn't been done. I don't think that it will be a huge issue though. I am thinking of 5 items at the bottom (same as on the side) and the rest is the same—no pop-up menus.
- The explorer links organisation need some thought. I agree that some links should be moved to other sections.
- I was thinking to have auto-albums as a "tab" or something similar in the albums menu.

I am still exploring options on how to organise links.

Commit:
Added translation using Weblate (Finnish)

Commit:
Translated using Weblate (Finnish)

Currently translated at 36.8% (152 of 412 strings)

Translation: Librephotos/LibrePhotos Frontend
Translate-URL: https://hosted.weblate.org/projects/librephotos/librephotos-frontend/fi/

derneuere committed:
Migrate login page to mantine

derneuere committed:
Use locale for login

derneuere committed:
Remove label

Nerodacles committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (412 of 412 strings)

Translation: Librephotos/LibrePhotos Frontend
Translate-URL: https://hosted.weblate.org/projects/librephotos/librephotos-frontend/es/

sickelap committed:
use rounded hover

sickelap commented:
I will reopen PR for navigation rework when other related bits will be done
